Nodejs/Product/Nodejs/NodejsTools.vsct

```diff
@@ -91,6 +91,13 @@
               <CommandFlag>DynamicVisibility</CommandFlag>
           If you do not want an image next to your command, remove the Icon node or set it to <Icon guid="guidOfficeIcon" id="msotcidNoIcon" /> -->
 
+       <Button guid="guidNodeToolsCmdSet" id="cmdidSendFeedback" priority="0x0500" type="Button">
+        <Parent guid="guidSHLMainMenu" id="IDG_VS_HELP_SUPPORT"/>
+        <Strings>
+          <ButtonText>Report Node.js Tools Issue or Send Feedback</ButtonText>
```

Comment:
a couple things:
1. text should be shorter maybe even just "Report Issue or Send Feedback" (most products actually split it into two items "Report Issue" and "Request Feature", so I have a hunch that that's more effective for some reason, but I don't know from experience)
2. Add a "..." at the end - this signifies that the action does not "complete" once you click, and is consistent with our UI paradigms throughout VS